docs/reference/docs/multi-get.asciidoc

```diff
@@ -45,28 +51,32 @@ GET /test/_mget
     ]
 }
 --------------------------------------------------
-//CONSOLE
+// TEST[setup:twitter]
 
-In which case, the `ids` element can directly be used to simplify the
-request:
+In this case, you can use the `ids` element to simplify the request:
 
 [source,console]
 --------------------------------------------------
-GET /test/_mget
+GET /twitter/_mget
 {
     "ids" : ["1", "2"]
 }
 --------------------------------------------------
+// TEST[setup:twitter]
+
+The response includes a `docs` array that contains the documents in the order specified in the request. 
+The structure of the returned documents is similar to that returned by the <<docs-get,get>> API.
```

Comment:
I think you need another line break to create a separate paragraph.
```suggestion

The structure of the returned documents is similar to that returned by the <<docs-get,get>> API.
```

Reply:
It doesn't need to be a new paragraph.